File: unit1/mario.c
// use for loops
// use boolean expressions

#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    do
    {
        height = get_int("How tall? ");
    }
    while (height <= 0 || height > 8);

    for (int i = 0; i < height; i++)
    {
        for (int h = height; h > i + 1; h--)
        {
            printf(" ");
        }

        for (int j = 1; j <= i + 1; j++)
        {
            printf("#");
        }

        printf("\n");
    }
}

File: unit6-python/retakes/mario.py
from cs50 import get_int

while True:
    height = get_int("How tall is the tower ")
    if height > 0 and height < 9:
        break

for i in range(0, height, 1):
    for f in range(0, height, 1):
        if i + f < height - 1:
            print(f" ", end="")
        else:
            print(f"#", end="")
    print(f"")
